app/assets/stylesheets/application.scss

```diff
@@ -15,10 +15,72 @@
  *= require foundation_and_overrides
  */
 
+ @import url(https://fonts.googleapis.com/css?family=Open+Sans:400,300,600,700,400italic,300italic);
+
+ @font-face {
+	font-family: 'icomoon';
+	src:url('../fonts/icomoon/icomoon.eot');
+	src:url('../fonts/icomoon/icomoon.eot?#iefix') format('embedded-opentype'),
+		url('../fonts/icomoon/icomoon.woff') format('woff'),
+		url('../fonts/icomoon/icomoon.ttf') format('truetype'),
+		url('../fonts/icomoon/icomoon.svg#icomoon') format('svg');
+	font-weight: normal;
+	font-style: normal;
+}
+
  /************** BASIC STYLING *****************************/
 
 #content {
-  background-color: #DADADA;
+  background-color: #FFF;
+  margin-top: 6rem;
+  margin-bottom: 6rem;
+}
+
+#background {
+  display: flex;
+  flex-flow: column;
+  height: 100%;
+  background: url(http://freephotos.atguru.in/hdphotos/forest-landscape-wallpaper/forest-sun-wallpaper-landscape-nature-wallpaper-2560-1600-widescreen-1080.jpg) no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+  overflow: auto;
+}
+
+/************** FOUNDATION ADJUSTMENTS *******************************/
+
+// .fullWidth {
+//    width: 100%;
+//    margin-left: auto;
+//    margin-right: auto;
+//    max-width: initial;
+// }
+
+/************** NAVIGATION *******************************/
+
+#nav-container {
+  border-bottom: 1px solid #d9d9d9;
+  box-shadow: 2px 0 5px rgba(0, 0, 0, 0.2);
+}
+
+.dropdown {
+  border: 1px solid #d9d9d9;
+  border-top: none;
+  box-shadow: 0 4px 5px 0 rgba(0, 0, 0, 0.2);
+}
+
+.dropdown li > a {
```

Comment:
Merge rule `.dropdown li > a` with rule on line 67<br>Selector should have depth of applicability no greater than 2, but was 3